pkg/datasource/compat/wrapper.go

```diff
@@ -120,37 +120,55 @@ func WrapAccessors(source datasource.DataSource, mntnsidAccessor datasource.Fiel
 		NetnsidAccessor: netnsidAccessor,
 	}
 
-	k8s, err := source.AddField("k8s", datasource.WithFlags(datasource.FieldFlagEmpty))
+	k8s, err := source.AddField("k8s", api.Kind_Invalid, datasource.WithFlags(datasource.FieldFlagEmpty))
```

Comment:
It feels a bit weird to actively use `api.Kind_Invalid` here - maybe we should add a dedicated `.AddContainer()` that sets the type to `Kind_Invalid` and also the `FieldFlagEmpty` in a later PR.

Reply:
Opened #2817 to keep track of this.